l10n_es_partner/models/res_partner.py

```diff
@@ -10,26 +10,33 @@ class ResPartner(models.Model):
     _inherit = "res.partner"
 
     comercial = fields.Char("Trade name", size=128, index="trigram")
-    display_name = fields.Char(compute="_compute_display_name")
+    complete_name = fields.Char(compute="_compute_complete_name")
```

Comment:
Por qué cambiar el campo?

Reply:
En el modelo res_partner en odoo 17 el campo ahora se llama complete_name

https://github.com/odoo/odoo/blob/579ee6d9792050955fa80346fd57ad294efcdd62/odoo/addons/base/models/res_partner.py#L194

En este pull 
https://github.com/odoo/odoo/pull/122085

dice:

For res.partner, rename the field store display_name into complete_name because display_name context-dependent and it makes no sense to have a compute store that is context-dependent.

Hasta la v16 si era display_name pero en la 17 el campo display_name ya no existe en partner

¿qué te parece?

Reply:
Finalmente no ha hecho falta cambiar el campo, olvida lo que dije anteriormente, no se por qué llegué a esa conclusión...